src/Apps/Artist/Routes/Consign/Utils/getConsignmentData.ts

```diff
@@ -37,8 +41,9 @@ export function getConsignmentData(pathname: string): ArtistConsignment {
       const recentlySoldArtworkIDs = flatten(
         artworks.map((artwork: any) => {
           const id = artwork["Artwork ids (recently sold) (comma separated)"]
-          // FIXME: Add proper type
-          artwork.artworkID = id
+          const realizedPrice = artwork["Realized Price (in dollars)"]
+          artwork.internalID = id
+          artwork.realizedPrice = realizedPrice
```

Comment:
Related to comment above

Reply:
I'm curious to know how you were thinking about types here. I see a note about a proper type, but then it looks like you opted for an unnamed type - any particular reason for that? No change necessary, just wondering!

Reply:
The idea here is to not mess with the original data too much, but also dont want to access data with these really long keys, so I added an alias on the object and then added some types to indicate useage. The unnamed types permit everything to remain how it is and not clutter up the interface with noise, and the additions that I made were made because they're used in code, and we want intellisense. 

Basically, just striking a balance between the CSV / excel table and the minimum amount of code required to parse / use it in a friendly way.